### DISCUSSION TITLE
Добавление украинского языка для языковых оповещений системы.

### DISCUSSION THREAD

**Discussion by plastilincheg** [mikopbx/Core#414] [Ideas (Идеи)]
Здравствуйте. Как можно добавить самостоятельно или при следующем обновлении украинские файлы для оповещений системы? Есть [тут ](https://github.com/vikdon/asterisk-core-sounds-uk) и форкнуто [тут](https://github.com/lexxai/asterisk-core-sounds-uk)

**Comment by boffart** [comment]
В новый релиз мы не можем добавить эти файлы, не ясно под какой лицензией они распространяются. 
Вручную добавить их вряд ли выйдет. не хватит места на разделе.

**Reply by plastilincheg** [reply]
Понятно. Я видел ваш запрос насчет лицензии у них от декабря 2021 года, думаю вам никто не ответит тк домен уже перекуплен и на нем совершенно другой сайт.

**Reply by boffart** [reply]
можно попробовать удалить не используемый набор звуковых файлов, к примеру 

`remount-offload`
`rm -rf /offload/asterisk/sounds/da-dk/;`

далее очистить к примеру каталог с английскими файлами:
`rm -rf /offload/asterisk/sounds/en-en/*`
(звезодчка на конце)

Скопируйте в него файлы для Украины
[Подключение к АТС с помощью WinSCP](https://wiki.mikopbx.ru/faq:winscp)

Затем в общих настройках указать язык для оповещений "Английский". 

Тут главное чтобы места хватило. Если не хватает, то нужно больше очищать, либо файлы копировать в gsm формате, они меньше места занимают.

**Reply by plastilincheg** [reply]
То есть удаляем языки до тех пор пока не хватит места? Или достаточно удалить датский и в его каталог загрузить, а после выбрать?

**Reply by plastilincheg** [reply]
Все получилось. Заменил датские звуковые файлы. 
/offload/asterisk/sounds/da-dk# du -lah
total 987K

**Reply by boffart** [reply]
гуд

**Reply by plastilincheg** [reply]
Собственник репо вышел со мной на связь и отписал, что добавил лицензию. Также уточнил если каких то звуковых файлов не хватает - написать ему.

**Reply by jorikfon** [reply]
А какая лицензия будет? Как с ним связаться? Мы можем в следующий релиз добавить эти звуковые файлы штатно в состав релиза.

**Reply by boffart** [reply]
https://github.com/vikdon/asterisk-core-sounds-uk/blob/master/LICENSE-asterisk-core-uk

**Reply by plastilincheg** [reply]
> А какая лицензия будет? Как с ним связаться? Мы можем в следующий релиз добавить эти звуковые файлы штатно в состав релиза.

admin@vikdon.com 

LICENSE
-------

Attribution-ShareAlike 4.0 International
